lit-base/model_service.py

```diff
@@ -0,0 +1,92 @@
+import mxnet as mx
+from model_loader import sym, arg_params, aux_params
+
+import tempfile
+import urllib2
+
+import numpy as np
+from PIL import Image
+from StringIO import StringIO
+
+
+ctx = mx.cpu()
+
+
+def preprocess(event):
+    """
+    Preprocess Lambda event into input format for mx mod.
+    :return: input NDArray
+    event : dict
+        Lambda event
+    """
+    img = None
+    try:
+        # GET method
+        if event.has_key('httpMethod') and event['httpMethod'] == 'GET':
+            url = event['queryStringParameters']['url']
+            # download image from url
+            img_file = tempfile.NamedTemporaryFile(delete=True)
+            if url:
+                req = urllib2.urlopen(url)
```

Comment:
consider using mx.test_utils.download(url) instead - it would avoid dependency on urllib